.eslintrc

```diff
@@ -33,6 +33,18 @@
           "rules": {
               "@scandipwa/scandipwa-guidelines/create-config-files": "off"
           }
+      },
+      {
+        "files": [
```

Comment:
I believe it's better to ignore all the `src/config/*`, not just these specific files.

Reply:
Make sense.